assets/shared/components/modal/index.js

```diff
@@ -30,7 +36,7 @@ const Modal = ( { onClose, title = '', children } ) => {
 	};
 
 	return createPortal(
-		<div className="sensei-modal">
+		<div className={ classnames( 'sensei-modal', className ) }>
```

Comment:
Can we avoid calling `classnames` here?

Maybe just having something like:

```suggestion
		<div className={ `sensei-modal ${className}` }>
```

instead?

I'm looking out mostly to avoid possible inconsistent usages of the `className` property here, so we don't end-up with calls to the `Modal` being called like:

```jsx
<Modal className={{"selected": someBooleanFlag}} ... />...</Modal>
```

Which...would be a little confusing, IMHO.